### PR TITLE
Update docs and state for junos_acl_interfaces

### PR DESCRIPTION
**Pull request by danielmellado**
This commit adds gathered state for junos_acl_interfaces and updates the
documentetion to the new resource modules versioning for collections.

### CODE REVIEW

**Comment by pabelanger** [comment]
recheck

File: plugins/modules/junos_acl_interfaces.py [hunk 1]
```diff
@@ -30,16 +30,12 @@
...
-    "supported_by": "network",
-}
+ANSIBLE_METADATA = {"metadata_version": "1.1", "supported_by": "network"}
 
 DOCUMENTATION = """
 ---
 module: junos_acl_interfaces
-version_added: "2.10"
+version_added: "1.0.0"
 short_description: Manage adding and removing Access Control Lists (ACLs) from interfaces on devices running Juniper JUNOS.
```
Comment (samccann) [inline plugins/modules/junos_acl_interfaces.py:39]:
```suggestion
short_description: Junos Access Control Lists (ACLs) interface resource module
```

**Comment by danielmellado** [comment]
recheck

**Comment by danielmellado** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Pull request merged**